common/tandemClient.js

```diff
@@ -198,6 +199,33 @@ export class TandemClient {
         return data;
     }
 
+    /**
+     * Reads elements from model using streaming.
+     * 
+     * @param {string} modelId - URN of the model.
+     * @param {string[]} [columnFamilies] - optional array of column families.
+     * @yields {Promise<object>} - async iterator of elements.
+     */
+    async *getElementStream(modelId, columnFamilies = [ ColumnFamilies.Standard ]) {
+        const token = this._authProvider();
+        const inputs = {
+            families: columnFamilies,
+            includeHistory: false,
+            skipArrays: true
+        };
+
+        const response = await fetch(`${this.basePath}/modeldata/${modelId}/scan`, {
+            method: 'POST',
+            headers: {
+                'Authorization': `Bearer ${token}`
+            },
+            body: JSON.stringify(inputs)
+        });
+        const elementStream = Readable.fromWeb(response.body).pipe(StreamArray.withParser()).pipe(new ElementFilter());
+
+        yield* elementStream;
```

Comment:
Using generators is probably less conventional choice here. But I see how it is convenient. I think a vast majority of applications won't benefit much from leveraging streaming capabilities here, given the complexity and general lack of streaming guarantees from our side, but it's a nice reference point.

Reply:
Since `scan` can return quite lot of elements we discussed streaming with @Blechhirn as possible alternative to pagination. An I thought it would be good to have example in case someone wants to try this.